src/docfx/lib/log/ErrorLog.cs

```diff
@@ -35,7 +35,7 @@ internal sealed class ErrorLog : IDisposable
 
         public IEnumerable<FilePath> ErrorFiles => _errorFiles;
 
-        public ErrorLog(string? outputPath, bool legacy = false)
+        public ErrorLog(string? outputPath = null, bool legacy = false)
```

Comment:
if `outputPath = null` is only for testing, can we remove the default value?

Reply:
sure